### ISSUE TITLE
What to use instead of gajira?

### ISSUE THREAD

**Issue by Piedone** [atlassian/gajira#18]
I see the project is not maintained anymore, neither the other gajira* actions. What should we use instead?

**Comment by babu-izhan-5001312** [comment]
Yeah, what should we use instead of this? I can't find any alternative way to create jira tickets using github actions ?

**Comment by timd73** [comment]
And unfortunately the existing actions will soon stop working, due to Node 16 end of life:

[gajira-login/issues/51](https://github.com/atlassian/gajira-login/issues/51)

[gajira-cli/issues/27](https://github.com/atlassian/gajira-cli/issues/27)

Has anyone found a supported alternative?

**Comment by d2callpod** [comment]
The deprecation message on this repo says we should use the API directly.

https://github.com/atlassian/gajira-cli

**Comment by appleboy** [comment]
I will implement the new Jira Action via API directly

**Comment by appleboy** [comment]
I create a new project for the Jira Data Center version. https://github.com/appleboy/go-jira

**Comment by appleboy** [comment]
See the new GitHub Action: https://github.com/appleboy/jira-action
